### PR TITLE
Update main.dart

### PR DESCRIPTION
**Pull request by salojoo**
Minor detail in commented code. Also the web tutorial had a similar issue, should provide Client instead of BrowserClient when using real API

### CODE REVIEW

**Review by thso — APPROVED** [review]

**Comment by kwalrath** [comment]
@chalin do we need to apply this in dart-lang/web-dev?

**Comment by chalin** [comment]
WAI. @thso please reverse the PR.

TLDR;

Hi all:

Thanks for the gesture of good will, but this was WAI, and this PR **should not** have been merged; see the comments in the original code, [dart-lang/site-webdev/examples/ng/doc/toh-6/web/main.dart#L10-L28](https://github.com/dart-lang/site-webdev/blob/master/examples/ng/doc/toh-6/web/main.dart#L10-L28).  I.e., this PR changes code that is in a comment and it was deliberately different from the running code.

As is mentioned in this repo's README, issues should be reported over at [dart-lang/site-webdev](https://github.com/dart-lang/site-webdev/issues/new?labels=example&title=%5BAngular%5D%5Bexample%5D%20tutorial/toh-6%3A%20). This applies to PRs too.  I'll see if I can disable that in the settings.

**Comment by chalin** [comment]
(Thanks for the notification Kathy. I had stopped watching these repos since I wasn't expecting any traffic.)

**Comment by thso** [comment]
You're right chalin@, misread what the PR was doing. Reverted.

Cheers
